apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/HHAClaimTransformerV2.java

```diff
@@ -35,6 +35,10 @@ final class HHAClaimTransformerV2 implements ClaimTransformerInterfaceV2 {
   /** The {@link NPIOrgLookup} is to provide what npi Org Name to Lookup to return. */
   private final NPIOrgLookup npiOrgLookup;
 
+  /** The metric name. */
+  private static final String METRIC_NAME =
+      MetricRegistry.name(HHAClaimTransformerV2.class.getSimpleName(), "transform");
+
```

Comment:
`TIMER_NAME` would be more informative here?  Or even `TRANSFORMATION_TIMER_NAME`.

Reply:
This is the metric name (structured like a path) that gets used when reporting out to New Relic. It is _also_ the timer's id, but primarily its supposed to be the NR id path.